css/styles.css

```
@@ -1,122 +1,238 @@
-* {
-	background-color: lightgrey;
-}
-/********************* CLASS SELECTORS ****************/
-.type-of-game {
-	text-align: center;
-	font-size: 80px;
-	font-family: cursive;
-	margin-top: 80px;
-	margin-bottom: 20px;
-}
-
-.name-of-game {
-  text-align: center;
-  font-size: 60px;
-  font-family: cursive;
-  margin-top: 10px;
-  margin-bottom: 50px;
-}
-
-.game-conteiner {
-	text-align: center;
-}
-
-.game-task1 {
-	text-align: center;
-	font-size: 50px;
-	font-family: sans-serif;
-	margin-bottom: 50px;
-}
-/********************* BUTTONS ****************/
-.blue-button {
-  -webkit-border-radius: 26;
-  -moz-border-radius: 26;
-  border-radius: 26px;
-  font-family: Georgia;
-  color: #ffffff;
-  font-size: 45px;
-  background: #3498db;
-  padding: 15px 25px 15px 25px;
-  text-decoration: none;
-  margin-left: 5px;
-  position: relative;
-  right: 50px;
-  
-}
-
-.blue-button:hover {
-  background: #3cb0fd;
-  background-image: -webkit-linear-gradient(top, #3cb0fd, #3498db);
-  background-image: -moz-linear-gradient(top, #3cb0fd, #3498db);
-  background-image: -ms-linear-gradient(top, #3cb0fd, #3498db);
-  background-image: -o-linear-gradient(top, #3cb0fd, #3498db);
-  background-image: linear-gradient(to bottom, #3cb0fd, #3498db);
-  text-decoration: none;
-}
-
-.red-button {
-  -webkit-border-radius: 26;
-  -moz-border-radius: 26;
-  border-radius: 26px;
-  font-family: Georgia;
-  color: #ffffff;
-  font-size: 45px;
-  background: #d93434;
-  padding: 15px 25px 15px 25px;
-  text-decoration: none;
-  margin-left: 5px;
-  position: relative;
-}
-
-.red-button:hover {
-  background: #fc3c3c;
-  text-decoration: none;
-}
-
-.green-button {
-  -webkit-border-radius: 26;
-  -moz-border-radius: 26;
-  border-radius: 26px;
-  font-family: Georgia;
-  color: #ffffff;
-  font-size: 45px;
-  background: #34d934;
-  padding: 15px 25px 15px 25px;
-  text-decoration: none;
-  margin-left: 5px;
-  position: relative;
-  left: 50px;
-}
-
-.green-button:hover {
-  background: #3cfa58;
-  text-decoration: none;
-}
-
-/********************* FOOTER ****************/
-
-footer {
-	margin-top: 50px;
-}
-
- p { 
- 	margin-top: 10px;
-	text-align: center;
-}
-
-.refButton {
-  -webkit-border-radius: 26;
-  -moz-border-radius: 26;
-  border-radius: 26px;
-  font-family: Georgia;
-  color: black;
-  font-size: 40px;
-  background: yellow;
-  padding: 15px 25px 15px 25px;
-  text-decoration: none;
-  display: block;
-  margin-top: 50px;
-  margin-left: auto;
-  margin-right: auto
-}
+body{
+  background-color: whitesmoke;
+  text-align: center;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+/********************* CLASS SELECTORS ****************/
+.name-of-game {
+  text-align: center;
+  font-size: 60px;
+  font-family: cursive;
+  margin-top: 1%;
+  margin-bottom: 1%;
+  text-shadow: 3px 0 #DA0463;
+}
+
+.game-conteiner {
+	text-align: center;
+}
+
+.game-task {
+	text-align: center;
+	font-size: 32px;
+	font-family: Georgia;

```

better to set this style for body tag and additional fonts.

```suggestion
         body {
           font-family: Georgia, sans-serif;
         }
```

---

Yes, thank you) I will change it now)